psychopy/app/builder/builder.py

```diff
@@ -2111,7 +2111,7 @@ def renameRoutine(self, name, event=None, returnName=True):
             # namespace:
             name = exp.namespace.makeValid(
                 name, prefix='routine')
-            if oldName in self.exp.routines.keys():
+            if oldName in list(self.exp.routines.keys()):
```

Comment:
I don't understand these changes, could someone explain? Why the explicit conversion?